### PR TITLE
React-Week1

### PR DESCRIPTION
**Pull request by Dima-Kutaini**

### CODE REVIEW

File: react/react1/week1/my-todo/src/component/toDo.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ .list{
```
Comment (LucasIversen) [inline react/react1/week1/my-todo/src/component/toDo.css:1]: Good to see you try out some styling in react native

File: react/react1/week1/my-todo/src/component/toDo.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+
+  export const TodoList = [
```
Comment (LucasIversen) [inline react/react1/week1/my-todo/src/component/toDo.jsx:4]: I'd add id's to the todos, to better mimic what you'd get from a server or API

File: react/react1/week1/my-todo/src/component/toDo.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      deadline: 'Fri Sep 15 2017',
+    },
+  ];
+
+
+export function TodoItem (props) {
+  return (
+    <div>
+    {props.TodoList.map((todo)=>{
+ return(
```
Comment (LucasIversen) [inline react/react1/week1/my-todo/src/component/toDo.jsx:24]: This probably should be it's own component called todoItem or something like that

**Review by LucasIversen — COMMENTED** [review]
Hi Dima. Good job. The only change I'd recommend you make is to put the render of a single todo in its own component 👍

File: react/react1/week1/my-todo/src/component/toDo.jsx (hunk 3 above)
Comment (Dima-Kutaini) [reply react/react1/week1/my-todo/src/component/toDo.jsx:24]: thanks for the review Lucas

**Pull request merged**